lang/R/inst/extdata/airr-schema.yaml

```diff
@@ -38,11 +38,11 @@ CURIEMap:
     CHEBI:
         type: ontology
         default:
-          map: OBO
-          provider: OLS
+            map: OBO
```

Comment:
There are several cases where there is an indentation of 2 is used instead of 4. This lack of consistency can cause some validation to fail. For example, in the case of the MHC enum on line 1273. A long term solution is to enforce yamls are written consistently in the github action checks to prevent this.